api/utils/keys/yubikey.go

```diff
@@ -51,10 +51,10 @@ const (
 
 var (
 	// We use slot 9a for Teleport Clients which require `private_key_policy: hardware_key`.
-	pivSlotNoTouch = piv.SlotAuthentication
+	PivSlotNoTouch = piv.SlotAuthentication
```

Comment:
nit:

```suggestion
	PIVSlotNoTouch = piv.SlotAuthentication
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms